src/types.ts

```diff
@@ -70,8 +70,8 @@ export interface TransactionEvent {
         | 'TransferredWithSchedule'
         | 'CredentialsUpdated'
         | 'DataRegistered'
-        | 'Interrupted'
-        | 'Resumed'
+        // | 'Interrupted'
```

Comment:
These should not just be commented out. Remove entirely if they are no longer relevant here.